File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.cs
using System;
using System.Linq;

public class Solution {
    public int solution(int num, int k) {
        
        var charArray = num.ToString().ToCharArray();
        int answer = Array.IndexOf(charArray, (char)(k + '0'));
        
        if(answer != -1)
            answer ++;
        
        return answer;
    }
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int num, int k) {
    
    string str = to_string(num);
    for (int i = 0; i < str.size(); ++i)
    {
        int n = str[i] - '0';
        if (n == k)
            return i + 1;
    }
    
    return -1;
}